src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = Lottery.getRandomBall();
+            System.out.println(balls[i].toString());
```

Comment:
toString will be called by default under the hood
combine if body into one line